README.rst

```diff
@@ -37,127 +37,117 @@ the Rescorla-Wagner learning rule to these corpora.
     This package is not intended to be used under python2.
 
 
-Installation
-============
-Install with::
+Getting Started
+===============
+These instructions will get you a copy of the project up and running on your
+local machine for development and testing purposes. See deployment for notes on
+how to deploy the project on a live system.
+
+
+Prerequisites
+-------------
+You need python 3.4 or newer and git installed on your machine. We recommend to
+install Minicoda (https://conda.io/miniconda.html) before installing ``pyndl``
+or to create a virtualenv within your personal folder.
+
+
+Installing
+----------
+If you only want to use the package run you can install ``pyndl`` from pypi with:
 
-    python setup.py install [--user]
+.. code:: bash
 
-or development install with::
+    pip install --user pyndl
```

Comment:
In (Mini/Ana)conda it is ``pip``, under Arch it seems to be pip as well?! (nevertheless ``pip3`` should work in most cases anyway).